code/KotobaUITests/KotobaUITests.swift

```diff
@@ -141,12 +140,5 @@ class KotobaUITests: XCTestCase
 		table.cells.element(boundBy: 1).swipeLeft()
 		app.buttons["Delete"].tap()
 		assertTableContents("three", "one")
-
-		// Edit mode to delete
-		app.buttons["Edit"].tap()
-		table.buttons["Delete one"].tap()
-		table.buttons["Delete"].tap()
-		app.buttons["Done"].tap()
-		XCTAssert(table.cells.count == 1)
```

Comment:
I think this was old code because the history view no longer has an edit button.